radixdlt-engine/src/main/java/com/radixdlt/application/system/state/EpochData.java

```diff
@@ -64,4 +64,22 @@
 
 package com.radixdlt.application.system.state;
 
-public record EpochData(long epoch) implements SystemData, HasEpochData {}
+import static com.radixdlt.atom.REFieldSerialization.*;
+
+import com.radixdlt.atom.SubstateTypeId;
+import com.radixdlt.atomos.SubstateDefinition;
+
+public record EpochData(long epoch) implements SystemData, HasEpochData {
```

Comment:
Can't we just inherit `SubstateDefinition`?

Reply:
Don't get what you mean. Make `EpochData` inherit `SubstateDefinition`?

Reply:
Yes, but probably there's no reason to make bigger refactoring now.